var/spack/repos/builtin/packages/pocl/package.py

```diff
@@ -56,7 +56,7 @@ class Pocl(CMakePackage):
     # (see #1616)
     # These are the supported LLVM versions
     depends_on("llvm +clang @6.0:11.0", when="@master")
-    depends_on("llvm +clang +shared_libs @6.0:11.0", when="@1.6")
+    depends_on("llvm +clang +shared_libs -flang @6.0:11.0", when="@1.6")
```

Comment:
@h-denpo Where did you try to build? I faced similar issues in other packages but apparently, at least in my case, it's just an issue with the resources needed to build LLVM on my laptop (8GB are not enough with flang).

Reply:
My system is a64fx with 32GB memory and haswell with 724GB memory.
On both systems,I can't build "spack install llvm%gcc +shared_libs+flang".
The llvm recipe has been changed to default to true for the flang variant.
 After that, it is no longer possible to build with +shared_libs without adding -flang.

Reply:
> My system is a64fx with 32GB memory and haswell with 724GB memory.

:sweat_smile: I think we ruled out memory issues!